http.h

```diff
@@ -1,40 +1,39 @@
 #ifndef __http_h
 #define __http_h
 
-#include <ev.h>
-#include <curl/curl.h>
+#include "nsq.h"
```

Comment:
`nsq.h` contains `include "http.h"`, and `http.h` contains `include "nsq.h"` - that's not good. Includes should not be circular, and also they should be avoided where not needed. You don't want it so that whenever you change one header, an incremental build needs to rebuild everything.

Reply:
Thanks for your patient review, I will check this kind of problem totally.